src/lib/y2storage/clients/inst_prepdisk.rb

```diff
@@ -79,6 +79,10 @@ def commit
         mount_in_target("/proc", "proc", "-t proc")
         mount_in_target("/sys", "sysfs", "-t sysfs")
         mount_in_target(EFIVARS_PATH, "efivarfs", "-t efivarfs") if mount_efivars?
+        devicegraph = Y2Storage::StorageManager.instance.staging
+        if devicegraph.encryptions&.any? { |e| e.method.id == :systemd_fde }
```

Comment:
NP: `Devicegraph#encryption` always returns an array, so safe navigation operator (`&`) is not needed.

```suggestion
        if devicegraph.encryptions.any? { |e| e.method.id == :systemd_fde }
```

Reply:
Fixed. Thanks !